### ISSUE TITLE
f5.bigip.ManagementRoot adds handler to root logger

### ISSUE THREAD

**Issue by SneakyPanda7** [F5Networks/f5-common-python#1523]
```
[~]> python
Python 3.7.0 (default, Oct  2 2018, 09:20:07)
[Clang 10.0.0 (clang-1000.11.45.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import logging
>>> logging.getLogger().handlers
[]
>>> import f5.bigip
>>> logging.getLogger().handlers
[]
>>> mgmt = f5.bigip.ManagementRoot('REDACTED', 'REDACTED', 'REDACTED')
>>> logging.getLogger().handlers
[<StreamHandler <stderr> (NOTSET)>]
>>>
```

If I'm writing a framework that uses f5.bigip, all my log messages go to console whether I like it or not. It appears that every method call I make using f5.bigip attaches this handler to the root logger.

**Comment by jasonrahm** [comment]
Hi @SneakyPanda7, pretty sure this issue belongs to the underlying icontrol rest library, but I'll investigate more closely on Monday.

**Comment by jasonrahm** [comment]
@SneakyPanda7, try installing [my pull request](https://github.com/F5Networks/f5-icontrol-rest-python/pull/168) as your f5-icontrol-rest-python package and see if that fixes your issue.

**Comment by SneakyPanda7** [comment]
@jasonrahm , yes that fixes it! Thanks. How soon can this get merged to master?

**Comment by jasonrahm** [comment]
I merged it to the 1.0 release branch, but I won't release until at least Friday. There are a couple other outstanding issues I'd like to address if possible.

**Comment by jasonrahm** [comment]
closing as merged in the icontrol-rest library
